src/sdk/namespace_fs.js

```diff
@@ -406,17 +466,22 @@ class NamespaceFS {
     /////////////////
 
     async read_object_md(params, object_sdk) {
+        let target_file;
```

Comment:
rename to file, doesn't make sense to call this target_file in operations where there is no source file
```suggestion
        let file;
```